koku/subs/trino_sql/azure_subs_pre_or_clause.sql

```diff
@@ -0,0 +1,36 @@
+SELECT
+  *,
+  COALESCE(date, usagedatetime) as subs_start_time,
+  date_add('day', 1, COALESCE(date, usagedatetime)) as subs_end_time,
+  json_extract_scalar(lower(additionalinfo), '$.vcpus') as subs_vcpu,
+  COALESCE(subscriptionid, subscriptionguid) as subs_account,
+  regexp_extract(COALESCE(resourceid, instancename), '([^/]+$)') as subs_resource_id,
```

Comment:
And here too I think
```suggestion
  regexp_extract(COALESCE(NULLIF(resourceid, ""), instancename), '([^/]+$)') as subs_resource_id,
```

Reply:
Added the `NULLIF`